pkg/perm/perm_test.go

```diff
@@ -1,4 +1,4 @@
-package str
+package perm
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [the file doesn't have an appropriate header](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)